src/pages/User/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Link, useNavigate } from "react-router-dom";
+import { googleLogout } from "@react-oauth/google";
+
+import "./User.scss";
+import { AppDispatch, RootState } from "@store/index";
+import { userLogOut } from "@store/auth/actions";
+import { updateBillingInfo } from "@store/cart/actions";
+import Header from "@containers/Header/Header";
+import Footer from "@containers/Footer/Footer";
+import PrimaryBtn from "@components/Buttons/PrimaryBtn/PrimaryBtn";
+import SecondaryBtn from "@components/Buttons/SecondaryBtn/SecondaryBtn";
+
+function User() {
+  const navigate = useNavigate();
+  const dispatch: AppDispatch = useDispatch();
+  const { user } = useSelector((state: RootState) => state.auth);
+
+  const handleClick = () => {
+    navigate("/wishlist");
+  };
+
+  const logOut = () => {
+    googleLogout();
+    dispatch(userLogOut());
+    dispatch(updateBillingInfo({ firstName: "" }));
+    dispatch(updateBillingInfo({ lastName: "" }));
+    dispatch(updateBillingInfo({ email: "" }));
```

Comment:
Think it makes sense to clean all fields that user filled in as they should be secured